src/Inciter/Transporter.h

```diff
@@ -195,7 +202,7 @@ class Transporter : public CBase_Transporter {
     void rowcomplete();
 
     //! Reduction target summing total mesh volume
-    void vol( tk::real v );
+    void totalvol( tk::real v );
```

Comment:
In general, we should consider a standardized way to indicate what is and isn't a reduction target. I think some of the stuff in transporter is ``redn_''?